### ISSUE TITLE
Allow pragma comments to allowlist certain issues

### ISSUE THREAD

**Issue by KevinHock** [bridgecrewio/checkov#508]
**Is your feature request related to a problem? Please describe.**
I want folks to be able to allowlist certain resources from certain issues, in the same way similar tools do this, such as pylint.

You would do this in case you wanted to bypass a check for a particular reason, or if there was a false-positive.

**Describe the solution you'd like**
In pylint you [can do](https://github.com/PyCQA/pylint/issues/1682#issuecomment-333307565) e.g. ` data = self._wrapper_class(data)  # pylint: disable=not-callable  # noqa`
whereas for HCL I would imagine
```
                "code_block": [
                    [
                        123,
                        "resource \"aws_s3_bucket\" \"some-vuln-s3\" # checkov: disable= CKV_AWS_18 {\n"
                    ],
```
I don't know if e.g. CloudFormation, and other things use for comments, but the feature is probably genericizable (🤔 how to spell that word properly).

**Describe alternatives you've considered**
A complementary feature, accomplishing something similar, would be: https://github.com/bridgecrewio/checkov/issues/379

**Comment by schosterbarak** [comment]
Thanks @KevinHock . is this answering the need: https://www.checkov.io/2.Concepts/Suppressions.html ?

**Comment by KevinHock** [comment]
Awesome, thanks @schosterbarak! Sorry I missed that, this solves that issue.
